index.html

```diff
@@ -49,129 +54,272 @@ <h4>I’m a Software Developer</h4>
-                    <img src="images/Illustration-header@2x.png" alt="Illustration header" class="hiden" width="350px">
+                    <img src="images/illustration-header-mobile.png" class="mobile" alt="Illustration header" width="240px">
+                    <img src="images/Illustration-header@2x.png" alt="Illustration header" class="hidden" width="360px">
                 </div>
             </section>
+            <!-- SECTION BASE END HERE -->
+
+            <!-- SECTION FOR THE MAIN STARTE HERE -->
 
             <section class="main">
-                <h2 class="top-title">My Recent Works</h2>
-                <div class="line"></div>
+                <div class="my-work">
+                    <h2 class="top-title">My Recent Works</h2>
+                    <div class="line"></div>
+                </div>
                 <div class="cards">
                     <div class="card">
-                        <a href="#"><img src="images/Img Placeholder.png" alt="project portfolio"></a>
-                        <h2>Multi-Post Stories</h2>
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                        <div class="row-skills">
-                            <ul>
-                                <li>CSS</li>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
+                        <div class="card-container-row">
+                            <div class="img-desktop">
+                                <a href="#" class="image-bg"><img src="images/Img Placeholder (1).png" alt="portfolio" class="hidden"></a>
+                                <a href="#"><img src="images/Img Placeholder (1).png" alt="project portfolio" class="mobile" width="350px"></a>
+                            </div>
+                            <div class="text-container-row">
+                                <h2>Multi-Post Stories</h2>
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                                <div class="row-skills">
+                                    <ul>
+                                        <li>CSS</li>
+                                        <li>HTML</li>
+                                        <li>Bootstrap</li>
+                                        <li>Ruby</li>
+                                    </ul>
+                                </div>
+                                <button type="submit">See project</button>
+                            </div>
```

Comment:
- [ ] Great work on incorporating the feedback from the previous reviewer! However, **I noticed a small issue with the technology elements in your project** that is causing it to deviate from the given template. It would be really helpful if you could take a moment to address this and make the necessary adjustments. 👌

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236360558-9ab9ef61-6853-4e05-ab36-8f172f342dc2.png)